sys-apps/ignition/files/0010-VMware-Fix-guestinfo.-.config.data-and-.config.url-v.patch

```diff
@@ -1,7 +1,7 @@
-From b4e062a912db68e50f5db35bdb7704889a5aedc8 Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Kai=20L=C3=BCke?= <kai@kinvolk.io>
-Date: Tue, 7 Apr 2020 17:36:35 +0200
-Subject: [PATCH 10/16] VMware: Fix guestinfo.*.config.data and *.config.url
+From d67ef76dcec2dafeba89a424c25c7f3f6e765bfa Mon Sep 17 00:00:00 2001
+From: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
+Date: Wed, 25 May 2022 10:34:32 +0200
+Subject: [PATCH 10/15] VMware: Fix guestinfo.*.config.data and *.config.url
```

Comment:
Did you change the authorship here by mistake?

Reply:
Oh yeah, applying failing patches is a PITA. Anyway, I remember seeing `Co-authored-by: Name Surname <email@stuff.com>` label. I think Github even understands it. It's used in the same way as `Signed-off-by` label.

Reply:
With `git am -3` it's sometimes possible to resolve the conflicts but at some point I also find it easier to redo the patch manually.

Reply:
yeah - that's why this patch is mentioned in the PR description but only 2 on the whole set of patches were annoying to apply, otherwise it went well. :)